lib/phoenix_component.ex

```diff
@@ -48,6 +48,21 @@ defmodule Phoenix.Component do
 
   You can learn more about the `~H` sigil [in its documentation](`Phoenix.LiveView.Helpers.sigil_H/2`).
 
+  For multiple dynamic attributes, the `{assigns}` syntax can be used.
```

Comment:
I think this is kind of a too early introduction to this. But it seems we really don't talk about this syntax here, so I think you were right, this should go to `sigil_H`. Sorry for the misleading info. :(